web/styles/compose.css

```diff
@@ -451,8 +451,40 @@
         bottom: 0;
     }
 
+    .upload_msg {
+        word-break: break-word;
+    }
+
+    .upload_banner_actions_button {
+        display: flex;
+        align-items: center;
+
+        .upload_banner_cancel_button {
+            border: none;
+            border-radius: 4px;
+            padding: 5px 10px;
+            font-weight: 600;
+            margin-left: 10px;
+            margin-top: 4.5px;
+            margin-bottom: 4.5px;
+            height: 32px;
+            white-space: nowrap;
+            background-color: hsl(204deg 63% 85% / 10%);
+            color: inherit;
+
+            &:hover {
+                background-color: hsl(204deg 63% 85% / 12%);
+            }
+
+            &:active {
+                background-color: hsl(204deg 63% 85% / 15%);
+            }
+        }
+    }
+
```

Comment:
Looking at these CSS changes, I don't believe these are at all necessary. If you look at a couple lines above, you'll find that the compose banners are styled based on the type of banner they are. (Alert, info, warning). One thing you'll notice is that the compose_banner_action_button for .info is missing, hence the missing style and color.

Reply:
Ah, I just noticed the class. Thanks for the heads-up. I have made the changes.